temba/flows/views.py

```diff
@@ -3377,3 +3435,316 @@ def get_context_data(self, *args, **kwargs):
             FlowStartCount.bulk_annotate(context["object_list"])
 
             return context
+
+
+class FlowTemplateForm(forms.ModelForm):
+    tags = forms.CharField(required=False)
+    orgs = forms.ModelMultipleChoiceField(
```

Comment:
The organization should be the one on the user session, I mean, the organization currently attached to the session.

Reply:
I think we can add a field (flag) when the template needs to be visible for all organizations.

Reply:
The`orgs` here is for visibility, so we can select organisation that should see a specific template  when they click on `Create Flow`. Organizations that deals with Health related stuff like `Covid-19 vaccine polls` have no business seeing templates meant for Elections. I can add a BooleanField as an override to show to all when its matters. What do you think?

Reply:
I think a BooleanField could work for this case. I have one concern with this approach for the orgs, because when we add a new organization, the templates marked as "Available for All Orgs", for example, should show up for this new org. Also, if we are going to have the "Template Group" field, should we have for the organization only flow templates related to the Health subject, or only related to Elections? I don't think so. I'm going to include @horkyborky on this thread, so he can take a look and approve if it's the way that he thought when modeling this feature for CCL.

Reply:
@musamusa @teehamaral Yes, all new orgs created should automatically have access to the flows that are marked for all orgs.  I agree with @teehamaral that a boolean probably works best to handle that.

I'm a little confused about template group and the functionality there.  My original thought was that someone from Implementations would go in and search for anything they want to add to a new account (that's not automatically added for all).  So let's say we have MusaCo, and they do surveys and elections.  Kate or whomever would search those two terms and add those flows to that new account.

Reply:
@horkyborky I believe a demo of the template feature can help clear some confusion. I can do a video demo and send it to you and @teehamaral  now or we can have a call for a demo. let me know which works best for you.

Reply:
It's up to you @horkyborky, but I think both call or a screen record works for me.